Local Testing Guide for Dagger-Registry Redirect.md

```diff
@@ -0,0 +1,86 @@
+# DAGGER-REGISTRY REDIRECT LOCAL TESTING
+
+This guide provides the steps to test the Dagger-Registry Redirect application locally, focusing on recent improvements to the syslog.
+
+### Run Vector Configuration Locally
+
+1. Install [Vector]((https://vector.dev/docs/setup/installation/)).
+2. Copy the following configuration and run it with Vector: vector --config vector.toml
+
+```toml file=vector.toml
+[sources.my_syslog_source]
+  type = "syslog"
+  mode = "tcp"
+  address = "0.0.0.0:12345"
+
+[transforms.my_filter]
+  type = "filter"
+  inputs = ["my_syslog_source"]
+  condition = '''
+message = string!(.message)
+contains(message, "dagger-registry-2023-01-23")
+'''
+
+[sinks.my_console_sink]
+  type = "console"
+  inputs = ["my_filter"]
+  target = "stdout"
+  encoding.codec = "json"
+```
+
+
+## Local Procedure
+
+Follow the steps below on different terminal instances:
+
+### Modifying the Hosts File
+
+Add the following line to your /etc/hosts file:
+
+```shell
+127.0.0.1 toto.localhost 
+```
+
+### Setting up Caddy Reverse Proxy
+
+1. Install [Caddy](https://caddyserver.com/docs/install)
+2. Run the following command to setup a reverse proxy:
+
+```shell
+caddy reverse-proxy --from toto.localhost --to http://localhost:8080
+```
+
+### Running the Main Application
+
+With the environment set up, run the Dagger-Registry Redirect application:
+
+```shell
+go run . -repo dagger
+```
+
+You should see the logs locally and in the Vector instance.
+
+To check the next logs, run the following Docker command (assuming the [equinix](https://github.com/orgs/dagger/packages/container/package/equinix-demo-day-2023) package exists). This command does not interfere with production as the registry-redirect is running locally and is separate from our Vector configurations.
+
+```shell
+docker pull toto.localhost/equinix-demo-day-2023:latest
+```
+
+## Fly Procedure
+
+### Deploy the app to FLY
+```shell
+cd magefiles && FLY_APP_NAME="dagger-registry-2023-07-28" FLY_API_TOKEN="$(flyctl auth token)" GITHUB_REF_NAME=main  go run ./main.go -w .. all
+```
+
+### Trigger logs
+
+To trigger the logs, run the following Docker command (assuming the [equinix](https://github.com/orgs/dagger/packages/container/package/equinix-demo-day-2023) package exists).
+
+1. Find the app name's hostname: https://fly.io/apps/dagger-registry-2023-07-28 in our case.
+
+2. Run below command:
+
+```shell
+docker pull dagger-registry-2023-07-28.fly.dev/equinix-demo-day-2023:latest
+```
```

Comment:
We should either capture this in dagger, or simply remove. It seems a lengthy process with various dependencies that may be reasonable to set up for development purposes. For others to test this locally, a mage target (a.k.a. daggerize) sounds much lower friction.

Reply:
Captured it here https://gist.github.com/grouville/e772c61a4213d51eac8cb9cabdea11f3